### DISCUSSION TITLE
Before and after table slots

### DISCUSSION THREAD

**Discussion by simple-hacker** [rappasoft/laravel-livewire-tables#312] [Feature Requests]
Would be able to get a before and after table slot to put extra layouts, and be able to store a modal in?

Thanks.

**Comment by rappasoft** [comment]
For what purpose? Why not just put the slots outside the component?

**Reply by simple-hacker** [reply]
I would like a confirmation modal for the bulk actions and individual rows.

Currently I have to store this modal outside of the datatable, and so when I perform an action on the table I have to emit up to show the modal, and then emit down again to perform the action, just too many emitting and listening.  Would be nice to store the modal in the datatable component itself as a slot or maybe even as the same as a filter/row views.

**Reply by rappasoft** [reply]
Ok i'll see where I can add a slot or two.

**Reply by rappasoft** [reply]
https://github.com/rappasoft/laravel-livewire-tables/commit/8d3e19deb15553c2ade435c422e0323be1749a02

Next release you can specify a view to be included within the component scope.

**Reply by dhruva81** [reply]
Hi 
Is this feature available ?
I want to put page heading title and  some buttons before a table. 
I am rendering the datatable on full page  without going through controller.
This will be helpful.

**Comment by lrljoe** [comment]
There are numerous slots now available, please advise if you need extra slots, and the proposed position(s)
